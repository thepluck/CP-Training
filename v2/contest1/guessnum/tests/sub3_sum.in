50
55
116
113
99
30
305
86
25
33
41
43
254
105
4
24
37
221
60
41
160
57
224
186
95
252
169
14
84
41
68
4
53
5
55
283
19
16
165
22
21
27
26
120
141
4
160
420
209
167
41
