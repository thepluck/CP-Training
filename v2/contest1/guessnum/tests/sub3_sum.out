610
2743
2587
1995
189
18999
1501
124
222
335
372
13171
2251
6
121
275
9955
741
335
5227
663
10246
7081
1828
13013
5817
42
1447
335
942
6
566
6
610
16361
73
53
5566
101
91
150
138
2950
4056
6
5227
36156
8896
5668
335
